src/runtime/tuple.c

```diff
@@ -52,6 +53,26 @@ void destruct_tuple(tuple t, boolean recursive)
     deallocate_tuple(t);
 }
 
+/* Compared to timm(), kern_timm() supports one key-value pair only. */
+tuple kern_timm(char *name, ...)
+{
+    tuple s = allocate_tuple();
+    assert(s != INVALID_ADDRESS);
+    vlist ap;
+    vstart(ap, name);
+    timm_term(s, name, &ap);
+    vend(ap);
+    return s;
+}
+KLIB_EXPORT_RENAME(kern_timm, timm);
```

Comment:
I created this kern_timm() function which is exported  as "timm" and is now used by the tls klib to create a status error.